README.md

```diff
@@ -195,38 +195,16 @@ This demonstrates:
 
```

Comment:
let's update the header to be `Webpack & React`?

Reply:
Updated! We may want to revisit ampersands vs plus signs later.

Reply:
ooo yeah plus sign is fine. im indifferent